spec/factories/funding_methods.rb

```diff
@@ -1,13 +1,13 @@
 # frozen_string_literal: true
 
 FactoryBot.define do
-  factory :bursary do
+  factory :funding_method do
```

Comment:
This needs to set the `funding_type` field. It can just set as a bursary for now.